systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaConnectorUtils.java

```diff
@@ -126,4 +126,24 @@ public static String waitForConnectorConfigUpdate(String podName, String connect
             () -> !oldConfig.equals(getConnectorConfig(podName, connectorName, apiUrl)));
         return getConnectorConfig(podName, connectorName, apiUrl);
     }
+
+    public static void waitForConnectorsSpecStability(String podName, String connectorName, String oldSpec) {
```

Comment:
Shouldn't be `spec` not used as string but as farbci8 `spec` ?

Why just spec? Shouldn't we check also generation for changes? Or spec is enough?

Reply:
I thought that it would be better to grep `spec` from the API. 

And idea of getting whole config is good, I'll change it. Thanks

Reply:
The `.spec` is actually changed only by the user (the STs here). The operator would change only the status. So it might be good to clarify what does it actually test and what needs to be stable.

Reply:
I wanted to test that when we change anything (in the `KafkaConnector` case the `taskMax`), the changes will not be applied if the reconciliations are paused. So, that's why I added there these stability "waits" and why I'm checking the specs. Maybe I can add some more comments to the tests.

Reply:
So ... it compares the REST API connector with the custom resource? I think that makes sense ... maybe the name of the method could be more explanatory and you can add some comment.

Reply:
Yeah @Frawless said completely same thing :smile: I'll change it, thanks :smile: